Fix templates -> examples and cover all our examples (#2163)
strimzi/strimzi-kafka-operator: documentation/assemblies/assembly-kafka-cluster.adoc

ifdef::Kubernetes[HostPath volumes on Minikube or]
Amazon EBS volumes in Amazon AWS deployments without any changes in the YAML files. The `PersistentVolumeClaim` can use a `StorageClass` to trigger automatic volume provisioning.

{ProductName} includes several examples for deploying a Kafka cluster.

> * `kafka-persistent.yaml` deploys a persistent cluster with 3 Zookeeper and 3 Kafka nodes.

Reviewer: ```suggestion
* `kafka-persistent.yaml` deploys a persistent cluster with three ZooKeeper and three Kafka nodes.
```

> * `kafka-jbod.yaml` deploys a persistent cluster with 3 Zookeeper and 3 Kafka nodes (each using multiple persistent volumes).

Reviewer: ```suggestion
* `kafka-jbod.yaml` deploys a persistent cluster with three ZooKeeper and three Kafka nodes (each using multiple persistent volumes).
```

> * `kafka-persistent-single.yaml` deploys a persistent cluster with 1 Zookeeper and 1 Kafka nodes.

Reviewer: ```suggestion
* `kafka-persistent-single.yaml` deploys a persistent cluster with a single ZooKeeper node and a single Kafka node.
```

> * `kafka-ephemeral.yaml` deploys an ephemeral cluster with 3 Zookeeper and 3 Kafka nodes.

Reviewer: ```suggestion
* `kafka-ephemeral.yaml` deploys an ephemeral cluster with three ZooKeeper and three Kafka nodes.
```

> * `kafka-ephemeral-single.yaml` deploys an ephemeral cluster with 3 Zookeeper and 1 Kafka nodes.

Reviewer: ```suggestion
* `kafka-ephemeral-single.yaml` deploys an ephemeral cluster with three ZooKeeper nodes and a single Kafka node.
```
